features/web-animations.yml

```diff
@@ -1,5 +1,5 @@
 name: Web animations
-description: The `animate()` method of `Element` objects programmatically animates elements over time and can synchronize the animations of multiple elements.
+description: The `animate()` method, together with the `Animation` interface, animates an element over time, possibly synchronizing the animations of multiple elements. The `getAnimations()` method queries and modifies current CSS animations, transitions, and web animations.
 spec: https://drafts.csswg.org/web-animations-1/
 caniuse: web-animation
 group: animation
```

Comment:
It's out of range of the diff, so I can't directly suggest it, but to match the description and fully resolve https://github.com/web-platform-dx/web-features/issues/2244 we also need to change which keys drive the headline status. This will also line us more closely with caniuse.

You'll need this in the current YAML file:

```
status:
  compute_from:
    - api.Element.animate
    - api.Animation
    - api.Document.getAnimations
```

and you'll need to run `npm install` and `npm run dist` and commit the results.